language-support/codegen-common/src/main/scala/com/digitalasset/daml/lf/codegen/Util.scala

```diff
@@ -1,17 +1,18 @@
 // Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package com.daml.codegen
+package com.daml.lf.codegen
 
-import com.daml.lf.iface.{Type, TypeConName, InterfaceType, EnvironmentInterface}
+import com.daml.lf.iface
+import com.daml.lf.iface.{EnvironmentInterface, InterfaceType, TypeConName}
 import com.daml.lf.data.Ref
 import scalaz.std.list._
 
 import scala.util.matching.Regex
 
 object Util {
 
-  private[codegen] def genTypeTopLevelDeclNames(genType: Type): List[Ref.Identifier] =
+  private[codegen] def genTypeTopLevelDeclNames(genType: iface.Type): List[Ref.Identifier] =
```

Comment:
Slightly more involved changes to avoid clashing with `Type` in the Java codegen. After this entire refactoring is done and after https://github.com/digital-asset/daml/pull/13366/ is merged we can reconsider how to deal with this.